docs/zh/latest/getting-started.md

```diff
@@ -62,15 +62,15 @@ $ curl --location --request GET "http://httpbin.org/get?foo1=bar1&foo2=bar2"
 
 让我们来分析一下上面的请求 URL：
 
-- Scheme: HTTP
+- Protocol: HTTP
```

Comment:
maybe we need to use `scheme` here?

Reply:
Hi @spacewander, do we need to keep the `scheme` here?

Reply:
As the English version uses Protocol, let's use Protocol here.